subiquity/client/controllers/progress.py

```diff
@@ -77,7 +77,8 @@ async def _wait_status(self, context):
             try:
                 app_status = await self.app.client.meta.status.GET(
                     cur=self.app_state)
-            except aiohttp.ClientError:
+            except Exception:
```

Comment:
catch Exception in a while True loop makes me nervous.
What if we at least log the cases where we caught an exception but it's not ClientError?
If someone gets stuck we should get a log file positively filled with whatever they are stuck on.

Reply:
yes that's fair